lib/astutils.h

```diff
@@ -30,6 +30,7 @@
 
 #include "config.h"
 #include "errortypes.h"
+#include "symboldatabase.h"
```

Comment:
hmm.. such a heavy dependency just for the `Scope::ScopeType` enum type. it is unfortunate. :-( But I don't see a good way around it.

Reply:
The ScopeType enum could be un-nested and forward-declared as `enum ScopeType : int`...